mbspotify/views_test.py

```diff
@@ -54,11 +54,6 @@ def create_app(self):
         ))
         return app
 
-    def test_index(self):
-        response = self.client.get("/")
-        # Index page should ask users to piss off.
-        self.assertIn("Piss off!", str(response.data))
```

Comment:
Instead this should test that template is being rendered without errors by comparing the HTTP status to 200.